src/scss/2_layout/_container.scss

```diff
@@ -188,13 +191,13 @@ aside {
     float: none;
     width: 100% !important;
   }
-}
+} // not sure where this comes from?
```

Comment:
This comes from the block of code at line 17 of media_overrides, but I can't remember why I thought it was a good idea to break the block up.